src/lib/radio/radio/radio.ts

```diff
@@ -107,21 +108,15 @@ const BaseRadioClass = WithFormAssociation(WithLabelAwareness(WithFocusable(With
 })
 export class RadioComponent extends BaseRadioClass implements IRadioComponent {
   public static get observedAttributes(): string[] {
-    return [
-      RADIO_CONSTANTS.attributes.CHECKED,
-      RADIO_CONSTANTS.attributes.DEFAULT_CHECKED,
-      RADIO_CONSTANTS.attributes.VALUE,
-      RADIO_CONSTANTS.attributes.DENSE,
-      RADIO_CONSTANTS.attributes.DISABLED,
-      RADIO_CONSTANTS.attributes.REQUIRED,
-      RADIO_CONSTANTS.attributes.READONLY,
-      RADIO_CONSTANTS.attributes.LABEL_POSITION,
-      RADIO_CONSTANTS.attributes.TABINDEX
-    ];
+    return Object.values(RADIO_CONSTANTS.observedAttributes);
   }
 
   private _foundation: RadioFoundation;
 
+  // Used to communicate with the form group after this radio instance has been disconnected
+  private _rootNode?: ShadowRoot | Document;
+  private _latestAssociatedForm?: HTMLFormElement;
```

Comment:
These seem fine to me. As long as we have tests in place to validate that these are properly updated when moving elements around in the DOM then I'm fine with it.